spec/models/patient_spec.rb

```diff
@@ -199,6 +201,91 @@ def refresh_views
 
         expect(described_class.ltfu_as_of(Time.current)).not_to include(not_ltfu_patient)
       end
+
+      describe "timezone-specific boundaries" do
```

Comment:
Do we need specs that include/exclude dead, migrated, transferred patients?

Reply:
Nope, verified that we test this in the control rate service.